reports-backend/src/main/java/org/jboss/da/reports/impl/LookupGeneratorImpl.java

```diff
@@ -69,17 +72,32 @@ public LookupGeneratorImpl(Configuration config) throws ConfigurationParseExcept
     @Override
     public Set<MavenLookupResult> lookupBestMatchMaven(Set<GAV> gavs, String mode, boolean brewPullActive)
             throws CommunicationException {
-        LookupMode lookupMode = getMode(mode);
+        LookupMode lookupMode = getMode(mode, false);
         ProductProvider productProvider = setupProductProvider(brewPullActive, lookupMode);
         Map<GA, CompletableFuture<Set<String>>> productArtifacts = getArtifactVersions(productProvider, gavs, true);
         return createLookupResult(gavs, lookupMode, productArtifacts);
     }
 
+    @Override
+    public Set<MavenVersionsResult> lookupVersionsMaven(
+            Set<GAV> gavs,
+            VersionFilter vf,
+            String mode,
+            boolean brewPullActive,
+            boolean includeBad) throws CommunicationException {
+        LookupMode lookupMode = getMode(mode, includeBad);
+        System.out.println("Lookup mode: " + lookupMode + " include bad?" + includeBad);
```

Comment:
I guess this is for debugging as it is just print and it's logged one level higher in LookupImpl?